src/playground/sample_data_tooling/requirements.txt

```diff
@@ -0,0 +1,17 @@
+attrs==21.4.0
```

Comment:
is this requirement.txt in root folder? is it correct?

Reply:
Yes it is correct. This may, however, be changed as I checkin other parts of my tooling.